### ISSUE TITLE
optimization loss

### ISSUE THREAD

**Issue by liyang1991c** [xingyul/flownet3d#40]
Hi,
Line 119 in train.py
`train_op = optimizer.minimize(loss, global_step=batch)`
Should this loss be loss[0]?  Since the loss is a tuple object from the get_loss() function.

Btw, was the pretrained model trained by a huber loss or L2 loss?

**Comment by xingyul** [comment]
`loss` is a scalar tensor. The pretrained model is trained under L2 loss.
